### ISSUE TITLE
Build errors: double and omitted imports

### ISSUE THREAD

**Issue by BenjaminGill-Metaswitch** [tupshin/cql-bindgen#5]
A portion of the build log is below (see https://travis-ci.org/BenjaminGill-Metaswitch/cassandra-sys-rs/builds/182538987). Nothing overly complicated - I'll have a go at fixing it up in the next few days.

@tupshin - what are you using to autogenerate this? Given the build warnings, it's possible that there's a bug that needs to be reported in the autogenerator, given that it doesn't seem to be producing valid Rust.

```
error[E0252]: a type named `cass_bool_t` has already been imported in this module
  --> src/lib.rs:32:9
   |
20 | pub use cassandra::cass_bool_t;
   |         ---------------------- previous import of `cass_bool_t` here
...
32 | pub use cassandra::*;
   |         ^^^^^^^^^^^^^ already imported

error[E0252]: a type named `CassError` has already been imported in this module
  --> src/lib.rs:32:9
   |
23 | pub use cassandra::CassError;
   |         -------------------- previous import of `CassError` here
...
32 | pub use cassandra::*;
   |         ^^^^^^^^^^^^^ already imported

error[E0432]: unresolved import `self::CassError_::*`
  --> src/error.rs:83:13
   |
83 |         use self::CassError_::*;
   |             ^^^^^^^^^^^^^^^^^^^^ Could not find `CassError_` in `error`

error[E0425]: unresolved name `CASS_ERROR_LIB_UNEXPECTED_RESPONSE`
  --> src/error.rs:97:55
   |
97 |                 Err(ErrorKind::LIB_REQUEST_QUEUE_FULL(CASS_ERROR_LIB_UNEXPECTED_RESPONSE).into())
   |                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ unresolved name

error[E0425]: unresolved name `CASS_ERROR_SERVER_WRITE_FAILURE`
   --> src/error.rs:200:52
    |
200 |                 Err(ErrorKind::SERVER_SYNTAX_ERROR(CASS_ERROR_SERVER_WRITE_FAILURE).into())
    |                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ unresolved name
```

**Comment by BenjaminGill-Metaswitch** [comment]
Ah - it's valid _nightly_-only Rust.

**Comment by tupshin** [comment]
fixed with 0.8.6 commit to work on stable

**Comment by tupshin** [comment]
sorry, to answer your actual question, i am using bindgen to build cassandra.rs, and just forgot that I was on nightly before that last change. my goal is to incorporate bindgen into the build process, but that's been challenging due to bindgen
